src/main/java/com/rocketden/main/service/GameManagementService.java

```diff
@@ -234,4 +236,16 @@ public void updateCode(String roomId, String userId, PlayerCode playerCode) {
     protected boolean isGameOver(Game game) {
         return game.getAllSolved() || (game.getGameTimer() != null && game.getGameTimer().isTimeUp());
     }
+
+    // Update people's socket active status
```

Comment:
I don't think so - this is the same logic as before but just refactored into a separate method to avoid duplicate code. Although that bug is very flaky so I don't actually know how often it occurs